internal/datacoord/import_scheduler.go

```diff
@@ -262,8 +245,11 @@ func (s *importScheduler) processInProgressImport(task ImportTask) {
 	}
 	resp, err := s.cluster.QueryImport(task.GetNodeID(), req)
 	if err != nil {
-		log.Warn("query import failed", WrapTaskLog(task, zap.Error(err))...)
-		s.checkErr(task, err)
+		updateErr := s.imeta.UpdateTask(task.GetTaskID(), UpdateState(datapb.ImportTaskStateV2_Pending))
```

Comment:
why set it back to Pending? just keep current state?

Reply:
datanode may crashed, need to retry this task